### ISSUE TITLE
Link profile page with navigation and side bar

### ISSUE THREAD

**Issue by Ngo-Tuan-Kiet** [hpi-swt2/bookkeeper-portal-red#82]
As a user I want to use the navigation or side bar to switch to my profile page, so that I can easily access my profile.

### Dependencies
- #28 
- #29 
- #42 
- #23 

### Acceptance Criteria
- [ ] profile icon is in navigation and side bar
- [ ] if I click on the icon I will be redirected to my profile page

Sidebar:
![Image](https://user-images.githubusercontent.com/66786380/204100623-50d50777-57f4-44b6-80a1-6b8972327ccf.png)

Navigation Bar in Home Screen:
![Image](https://user-images.githubusercontent.com/66786380/204100673-d2802514-a039-4821-b4dc-dcddb0c02322.png)

**Comment by Skn0tt** [comment]
We're blocked on https://github.com/hpi-swt2/bookkeeper-portal-red/issues/28 and https://github.com/hpi-swt2/bookkeeper-portal-red/issues/29. It looks like there's a [branch](https://github.com/hpi-swt2/bookkeeper-portal-red/commit/0cf4c72500502fcd9d725c518daaa45d79bb2561) where progress is being made. Once that's merged, we should be able to fulfil this issue by placing the link here:

https://github.com/hpi-swt2/bookkeeper-portal-red/blob/210d550f91f4c6769f322fb8f8f39067e7b91ddc/app/views/layouts/application.html.erb#L66

```diff
- <a href="#" class="text-white"><i class="bi bi-person-fill fs-3"></i></a> 
+ <a href="<%= profiles_me_path %>" class="text-white"><i class="bi bi-person-fill fs-3"></i></a> 
```

**Comment by yannik-dittmar** [comment]
This issue was already solved by #29.
